plugins/tasks/resource/pom.xml

```diff
@@ -25,6 +25,16 @@
             <artifactId>concord-runtime-sdk-v2</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.walmartlabs.concord</groupId>
+            <artifactId>concord-common</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
```

Comment:
Why does it need commons-compress now? IIRC, it only needed if you use `zip*` methods from concord-common.

Reply:
`commons-compress` is not needed by the resource task, but it _is_ a transitive dependency of `concord-common`. Since `concord-common` is `provided` scope, its transitive deps aren't loaded for the unit tests. It could probably just be `test` scope instead of provided here.